src/Akka.CQRS.Hosting.Tests/TradeProcessor.cs

```diff
@@ -0,0 +1,295 @@
+﻿
+
+using Akka.Actor;
+using Akka.Cluster.Hosting;
+using Akka.Hosting;
+using Akka.CQRS.Infrastructure;
+using Akka.CQRS.TradeProcessor.Actors;
+using Akka.Util;
+using Akka.Persistence.SqlServer.Hosting;
+using Akka.Remote.Hosting;
+using FluentAssertions;
+using Akka.CQRS.Pricing.Actors;
+using Akka.Persistence.Query;
+using Akka.Persistence.Query.Sql;
+using Petabridge.Cmd.Host;
+using Petabridge.Cmd.Cluster.Sharding;
+using Petabridge.Cmd.Remote;
+using Petabridge.Cmd.Cluster;
+using Akka.CQRS.Pricing.Cli;
+using Microsoft.Extensions.Hosting;
+
+namespace Akka.CQRS.Hosting.Tests
+{
+    public class TradeProcessor : Akka.Hosting.TestKit.TestKit
+    {
+        private readonly string _sqlConnectionString = "Server=sql,1633;User Id=sa;Password=This!IsOpenSource1;TrustServerCertificate=true";
+       
+       
+        private readonly ITestOutputHelper _output;
+        public TradeProcessor(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+        private async Task<IHost> TraderHost()
+        {
+            var tcs = new TaskCompletionSource();
+            using var cancellationTokenSource = new CancellationTokenSource(TimeSpan.FromSeconds(120));
+
+            var host = new HostBuilder()
+                .ConfigureServices(collection =>
+                {
+                    collection.AddAkka("AkkaTrader", builder  =>
+                    {
+                        #region Trade Processors
+                        builder
+                            .WithRemoting(new RemoteOptions

```

Comment:
So rather than copy and paste the code we use to startup each of these services inside our test library, we'd be better off if we developed some reusable, composable extension methods on top of Akka.Hosting - per the best practices we recommend here: https://www.youtube.com/watch?v=U2IIfLKxprM&t=1777s

Here's an example of how we do that inside our WebApiTemplates for instance:

https://github.com/akkadotnet/akkadotnet-templates/blob/8d89c75627b731b4e54758a1ed7b73083c9337d5/src/WebApiTemplate/src/WebApiTemplate.App/Configuration/AkkaConfiguration.cs#L42-L50

This makes it really use for us to only call the parts of each service that we actually need during testing, _re-using_ the exact same code we use to perform this configuration in production:

https://github.com/akkadotnet/akkadotnet-templates/blob/8d89c75627b731b4e54758a1ed7b73083c9337d5/src/WebApiTemplate/tests/WebApiTemplate.App.Tests/CounterActorSpecs.cs#L47-L57

That way, if we change how config works for production - config for tests == updated automatically.